src/azure-cli-testsdk/azure/cli/testsdk/preparers.py

```diff
@@ -3,108 +3,15 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-import inspect
-import functools
 import os
 
-from .base import ScenarioTest, execute
+from azure_devtools.scenario_tests.base import ScenarioTest
+from azure_devtools.scenario_tests.utilities import create_random_name
+from azure_devtools.scenario_tests.recording_processors import RecordingProcessor
+from azure_devtools.scenario_tests.preparers import AbstractPreparer, SingleValueReplacer
```

Comment:
Same here, the point was to keep compat with `from azure.cli.testsdk.preparers import AbstractPreparer`